bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/proposal/ProposalView.scala

```diff
@@ -407,8 +406,8 @@ class ProposalView(advisor:ShellAdvisor) extends BorderPanel with BoundView[Prop
 
   }
 
-  // public edit method for quick fix
-  def edit(i:Investigator) = investigators.listViewer.edit(i)
-
-
+  // public edit methods for quick fixes
+  def editPi(setup: PiEditor => Unit = _ => {}) = investigators.listViewer.editPi(setup)
+  def editCoi(i: CoInvestigator, setup: CoiEditor => Unit = _ => {}) = investigators.listViewer.editCoi(i, setup)
```

Comment:
It seems `editCoi` is never called with a non-default setup method, Do we need it?